<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Libmenu</class>
 <widget class="QWidget" name="Libmenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>907</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Libmenu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Libmenu {	
border-image: url(:/LIBPRO/pattern-5_blur.png);
}

QPushButton {	
	border-radius: 40px;
	background-color: qlineargradient(spread:reflect, x1:0.669951, y1:0.727273, x2:0, y2:0, stop:0 rgba(168, 237, 205, 255), stop:0.492611 rgba(255, 255, 255, 255));

}


QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 rgb(212, 236, 215));
}

QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ddd);
}


QLineEdit {
	border: 1px solid grey;
	border-radius: 7px;
    width: 200px;
    height: 150px;
}

QLabel {
	color: #646568;
	font-size: 14px;
	font-weight: bold;
	font-family: &quot;Bangla Sangam MN&quot;;
}</string>
  </property>
  <widget class="QPushButton" name="buttonLogout">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>10</y>
     <width>64</width>
     <height>60</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>power-button-md.png</normaloff>power-button-md.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>75</y>
     <width>1281</width>
     <height>831</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="All_book">
    <attribute name="title">
     <string>All book</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>1221</width>
       <height>701</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1221</width>
        <height>701</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </widget>
    <widget class="QLineEdit" name="searchLine">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>1091</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Calligraphy</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Search books</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>1160</x>
       <y>20</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
border-radius: 15px;
background: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>search_icon.png</normaloff>search_icon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>60</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Add_book">
    <attribute name="title">
     <string>Add book</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>410</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 20px;</string>
     </property>
     <property name="text">
      <string>Choose File</string>
     </property>
    </widget>
    <widget class="QLabel" name="book_Img">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>221</width>
       <height>341</height>
      </rect>
     </property>
     <property name="text">
      <string>*Book Image*</string>
     </property>
    </widget>
    <widget class="QLabel" name="type">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>130</y>
       <width>47</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
    <widget class="QLabel" name="date">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>130</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Date Released</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="typeEdit">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>120</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameEdit">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>50</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="authorEdit">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>50</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="name">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>120</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="author">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Author</string>
     </property>
    </widget>
    <widget class="QLabel" name="rating">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>200</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Rating</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="borrowEdit">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>180</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ratingEdit">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>180</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="number">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>200</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Max Borrowed</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="linkEdit">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>245</y>
       <width>671</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="link">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>250</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Link</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>500</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Review</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="summary">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>310</y>
       <width>651</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>310</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Sumary</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="review">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>490</y>
       <width>651</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>690</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Book_request">
    <attribute name="title">
     <string>Book requests</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>1221</width>
       <height>741</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1221</width>
        <height>741</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="aboutButton">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>0</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="LIBPRO.qrc">
     <normaloff>:/LIBPRO/Alecive-Flatwoken-Apps-Help-Info.ico</normaloff>:/LIBPRO/Alecive-Flatwoken-Apps-Help-Info.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LIBPRO.qrc"/>
 </resources>
 <connections/>
</ui>
